source/BleSerial.h

```diff
@@ -0,0 +1,244 @@
+/*
+    Copyright(c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
Probably not, but I'd rather keep them in for now, so we don't need to run it by LCA